src/main/java/vention/pages/BasePage.java

```diff
@@ -0,0 +1,73 @@
+package vention.pages;
+
+import vention.configLoader.ConfigLoader;
+import vention.driver.DriverManager;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.By;
+import java.time.Duration;
+
+public abstract class BasePage {
+  private static final int DEFAULT_TIME = 10;
+
+  public BasePage() {
+    PageFactory.initElements(DriverManager.getDriver(), this);
+  }
+
+  public String getRelativePath() {
```

Comment:
Why do you need initialization of this class in the BasePage class as it's specific for each page? 

You can keep this method but make it abstract, override in every page class.